### ISSUE TITLE
Changing column width in JabRef 5.0 prevents opening 4.3.1 #4579

### ISSUE THREAD

**Issue by benjagooder** [benjagooder/jabref#19]
Link to original bug issue: 
- https://github.com/benjagooder/jabref/projects/1#card-17298938

Latest development version has been tested from http://builds.jabref.org/master/ and the problem persists

Open JabRef 5.0 and add or change at least one of the column widths in Preferences -> Entry table columns.
Save preferences.
Open JabRef 4.3.1.
Open JabRef 4.3.1 Preferences window.
Expected behavior: the preferences window in JabRef 4.3.1 opens.

Actual behavior: the preferences window in JabRef 4.3.1 does not open. An error log in my terminal points out that every column width for the main table is saved in an unknown or wrong format (e.g "100.0", "32.0" etc).

Workaround: open JabRef 5.0, delete all entries in Entry table columns, open JabRef 4.3.1 again and the Preferences should work.

Why it matters: I've been testing version 5.0, but I still face some problems that force me to use version 4.3.1 when doing actual work. This bug makes it impossible to use both versions, unless there is a way to separate their preferences that I don't know about.

tl;dr: JabRef 5 treats the column width values of the main table as floats, JabRef < 5 treats them as integers. This makes opening the preferences in older JabRef after editing them in new JabRef impossible.
